### ISSUE TITLE
Запрос на добавление параметра для изменения позиции уведомлений в компоненте Toaster

### ISSUE THREAD

**Issue by a-abdurazzoq** [gravity-ui/uikit#1915]
Привет, команда!

В нашем проекте мы используем компонент Toaster из пакета @gravity-ui/uikit. В настоящее время отсутствует возможность изменять позицию уведомлений на экране.

Было бы здорово, если бы появился параметр, который позволял бы задавать позицию уведомлений. 
Предлагаемые позиции: `"top-left" | "top-center" | "top-right" | "bottom-left" | "bottom-center" | "bottom-right"`

Эта функция добавила бы больше гибкости при размещении уведомлений в различных частях экрана, в зависимости от требований интерфейса. Дополнительной кастомизации позиции не требуется, достаточно этих предустановленных вариантов.

Заранее спасибо!

**Comment by korvin89** [comment]
cc @ogonkov

**Comment by korvin89** [comment]
@a-abdurazzoq Do you actually need `"top-center"` and `"bottom-center"` positions? We are not quite sure about them

**Comment by a-abdurazzoq** [comment]
> @a-abdurazzoq Do you actually need `"top-center"` and `"bottom-center"` positions? We are not quite sure about them

No, I don't actually need those positions. I included them just in case someone else might find them useful.

**Comment by ykamendrovskiy** [comment]
Hey guys! From the design side all the corner-anchored options seem ok
cc @korvin89 @ogonkov
